.github/workflows/publish.yml

```diff
@@ -13,11 +13,15 @@ jobs:
         with:
           node-version-file: '.nvmrc'
           registry-url: 'https://registry.npmjs.org'
+          # Defaults to the user or organization that owns the workflow file
+          scope: '@waveshq'
       - name: Install dependencies
         run: npm ci
       - name: Build the package
         run: npm run build
       - name: Publish package on NPM 📦
-        run: npm publish --access public
+        run: npm publish --access public --tag latest
         env:
           NODE_AUTH_TOKEN: ${{ secrets.WAVESHQ_NPM_TOKEN }}
+      - run: npm config delete "//registry.npmjs.org/:_authToken" --location=project
```

Comment:
why need this?

Reply:
This deleted the authToken after the action is done in .npmrc file. Doesn't matter if successful or not.